tests/bulk/bulkwrite-update-006.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+MongoDB\Driver\BulkWrite::update() PackedArray for update pipeline
```

Comment:
This and the arrayFilters test that follows were ported from existing `bulkwrite-update` tests.